crates/std_detect/src/detect/os/windows/aarch64.rs

```diff
@@ -51,28 +57,23 @@ pub(crate) fn detect_features() -> cache::Initializer {
                 Feature::dotprod,
                 IsProcessorFeaturePresent(PF_ARM_V82_DP_INSTRUCTIONS_AVAILABLE) != FALSE,
             );
+            // jsconv requires float support
             enable_feature(
                 Feature::jsconv,
-                IsProcessorFeaturePresent(PF_ARM_V83_JSCVT_INSTRUCTIONS_AVAILABLE) != FALSE,
+                fp && IsProcessorFeaturePresent(PF_ARM_V83_JSCVT_INSTRUCTIONS_AVAILABLE) != FALSE,
```

Comment:
I don't think we need to be pedantic about things like this. It's impossible for an AArch64 cpu to support jsconv without fp.